plugins/modules/aws_kms.py

```diff
@@ -164,9 +164,9 @@
             type: dict
   policy:
     description:
-      - policy to apply to the KMS key
+      - policy to apply to the KMS key. Provide a valid JSON string.
```

Comment:
```suggestion
      - policy to apply to the KMS key.
```

Reply:
Not really necessary to add the 'provide a valid json string' if we define that it needs to be 'json' as the type of input.

Reply:
Just thought it would make things clearer anyway. But I don't have a problem with removing it.